install.sh

```diff
@@ -0,0 +1,3 @@
+conda env create --file environment.yml
```

Comment:
Probably good to add a header indicating this is a `bash` script (see, e.g. https://stackoverflow.com/q/8967902).